gradle/javaProject.gradle

```diff
@@ -191,10 +191,20 @@ check {
     dependsOn createJavadoc
 }
 
+def getCheckedOutGitCommitHash() {
```

Comment:
This was needed for a different approach we used for code coverage. We don't need the changes in this file